src/runtime/StaticStrings.h

```diff
@@ -50,6 +50,7 @@ namespace Escargot {
     F(Collator)                   \
     F(DataView)                   \
     F(Date)                       \
+    F(deref)                      \
```

Comment:
Please keep the alphabetical order (lower case is located after capital letters)